src/main/java/tppe/tp1/acesso/AcessoBuilder.java

```diff
@@ -43,6 +44,29 @@ public Float getValorAcesso() {
 	public Float getValorContratante() {
 		return valorContratante;
 	}
+	
+	public Integer getDiariasDiurnas(EstacionamentoBuilder e) {
```

Comment:
Se for pra retornar 0 ou  1 é melhor usar boolean

Se for usar boolean poe o nome da função de isDiariaDurna, pra ja passar a ideia de checagem true/false

Reply:
Feito :+1: